theano/gpuarray/tests/test_elemwise.py

```diff
@@ -346,7 +348,9 @@ def setUp(self):
 
 class T_gpureduce_dtype(test_elemwise.T_reduce_dtype):
     mode = mode_with_gpu.excluding('local_cut_useless_reduce')
-    op = GpuCAReduceCuda
+
+    # GpuDnnReduction doesn't cover all cases, but should cover some
+    op = (GpuCAReduceCuda, GpuDnnReduction)
```

Comment:
I think there should be a couple of tests to check that `GpuDnnReduction` is actually inserted at least at a couple of places.